sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ActionDescriptorFactory.scala

```diff
@@ -85,23 +109,40 @@ private[impl] object ActionDescriptorFactory extends ComponentDescriptorFactory
           .withKalixOptions(kalixOptions)
       }
 
-    def combineByTopic(subscriptions: Seq[KalixMethod]): Seq[KalixMethod] = {
+    val subscriptionTopicClass: Array[KalixMethod] =
+      if (hasTopicSubscription(component)) {
+        component.getMethods.sorted // make sure we get the methods in deterministic order
+          .filter(hasActionOutput)
+          .collect {
+            case method
+                if !subscriptionTopicMethods.exists(s =>
+                  s.serviceMethod.methodName == method.getName) => // individual annotated overrides class level annotation
```

Comment:
we don't need this check anymore, correct?

Reply:
correct